src/extended-typescript-service.ts

```diff
@@ -366,95 +363,59 @@ export class ExtendedTypescriptService extends TypeScriptService {
                     )
                     const info = configuration.getService().getQuickInfoAtPosition(fileName, offset)
                     if (!info) {
-                        return Observable.of({ contents: [] })
+                        return { contents: [] }
                     }
                     const contents: (MarkedString | string)[] = []
-
-                    // mode
-                    const definitions = configuration.getService().getDefinitionAtPosition(fileName, offset);
-
-                    const result = (defintionUri: string) => {
-                        const isDocAccessible = this.isUriAccessible(defintionUri);
```

Comment:
Why needn't `isDocAccessible ` any more

Reply:
previously we are checking is doc is accessible at runtime, but better way is to filter dependencies after it is downloaded so there is no chance it get loaded into memory